### PR TITLE
TN-351 formatting test_truenth.py, test_user.py, test_user_document.py

### PR DESCRIPTION
**Pull request by John0Randolph**

### CODE REVIEW

File: tests/test_user.py [hunk 1]
```diff
@@ -193,16 +189,16 @@ def test_australian_indigenous_status(self):
...
-                "coding": [
-                    {"system": TRUENTH_VALUESET_NHHD_291036,
-                     "code": "1"
-                    },
-                    {"system": TRUENTH_VALUESET_NHHD_291036,
-                     "code": "9"
-                    },
-                ]
-            }}
+                     "valueCodeableConcept": {
```
Comment (ivan-c) [inline tests/test_user.py]: Too much indentation, are you trying to align it with the above line?

File: tests/test_user.py [hunk 2]
```diff
@@ -427,15 +424,15 @@ def test_account_creation_by_staff(self):
             'organizations': [{'organization_id': org_id},
                               {'organization_id': org2_id}],
             'consents': [{'organization_id': org_id,
-                        'agreement_url': 'http://fake.org',
-                        'staff_editable': True,
-                        'send_reminders': False}],
+                          'agreement_url': 'http://fake.org',
```
Comment (ivan-c) [inline tests/test_user.py]: Too much indentation, are you trying to align it with the above line?

File: tests/test_user.py [hunk 3]
```diff
@@ -1134,17 +1135,17 @@ def test_password_reset(self):
...
         assert response.status_code == 200
 
     def test_bogus_user_contensent(self):
         # example bogus data from msk testing
         data = {
             'agreement_url':
-            'https://stg-lr7.us.truenth.org/c/portal/truenth/asset?editorUrl='
-            'true&version=1.6&groupId=20147&uuid=09bb5690-d49b-a10e-'
-            '5339-e677353e694f',
+                'https://stg-lr7.us.truenth.org/c/portal/truenth/asset?editorU'
```
Comment (ivan-c) [inline tests/test_user.py]: Can you split this on the `?` (not in the middle of a querystring parameter)?

File: tests/test_user.py (hunk 2 above)
Comment (ivan-c) [reply tests/test_user.py]: please don't indent more than one level

Comment (ivan-c) [reply tests/test_user.py]: Hmm still seeing it in the [diff](https://github.com/uwcirg/true_nth_usa_portal/pull/2515/files), give it another try?

File: tests/test_user.py [hunk 4]
```diff
@@ -611,15 +611,15 @@ def test_roles_duplicate_add(self):
...
+            {"name": ROLE.APPLICATION_DEVELOPER.value},
+        ]}
 
         self.promote_user(role_name=ROLE.ADMIN.value)
         self.promote_user(role_name=ROLE.APPLICATION_DEVELOPER.value)
         self.login()
         response = self.client.post('/api/user/%s/roles' % TEST_USER_ID,
-                content_type='application/json',
-                data=json.dumps(data))
+                                    content_type='application/json',
```
Comment (ivan-c) [inline tests/test_user.py]: Please don't indent more than one level at once

File: tests/test_user.py [hunk 5]
```diff
@@ -973,15 +974,15 @@ def test_subject_relationships(self):
 
     def test_set_relationships(self):
         other_user = self.add_user(username='other@foo.com')
-        data = {'relationships':[{'user': TEST_USER_ID,
-                                  'has the relationship': 'partner',
-                                  'with': other_user.id},]
-               }
+        data = {'relationships': [{'user': TEST_USER_ID,
+                                   'has the relationship': 'partner',
```
Comment (ivan-c) [inline tests/test_user.py]: Please don't indent more than one level at once

File: tests/test_user.py [hunk 6]
```diff
@@ -1022,15 +1023,15 @@ def test_delete_relationships(self):
         "delete now done by PUTting less than all relationships"
         self.create_fake_relationships()
         other_user = User.query.filter_by(username='other@foo.com').first()
-        data = {'relationships':[{'user': other_user.id,
-                                  'has the relationship': 'sponsor',
-                                  'with': TEST_USER_ID},]
-               }
+        data = {'relationships': [{'user': other_user.id,
+                                   'has the relationship': 'sponsor',
```
Comment (ivan-c) [inline tests/test_user.py]: Please don't indent more than one level at once

File: tests/test_user.py [hunk 7]
```diff
@@ -1160,7 +1161,7 @@ def test_bogus_user_contensent(self):
     def test_locale_inheritance(self):
         # prepopuate database with matching locale
         cd = Coding.from_fhir({'code': 'en_AU', 'display': 'Australian English',
-                  'system': "urn:ietf:bcp:47"})
+                               'system': "urn:ietf:bcp:47"})
```
Comment (ivan-c) [inline tests/test_user.py]: Please don't indent more than one level at once

File: tests/test_user.py (hunk 2 above)
Comment (John0Randolph) [reply tests/test_user.py]: @ivan-c still see it?

File: tests/test_user.py [hunk 8]
```diff
@@ -1022,15 +1025,16 @@ def test_delete_relationships(self):
...
         other_user = User.query.filter_by(username='other@foo.com').first()
-        data = {'relationships':[{'user': other_user.id,
-                                  'has the relationship': 'sponsor',
-                                  'with': TEST_USER_ID},]
-               }
+        data = {'relationships': [{
+            'user': other_user.id,
+            'has the relationship': 'sponsor',
+            'with': TEST_USER_ID}, ]
+                }
```
Comment (ivan-c) [inline tests/test_user.py]: Can you put this on the same line as the `]`?

File: tests/test_user.py [hunk 9]
```diff
@@ -811,14 +813,14 @@ def test_deep_tree_check_role(self):
         staff_top = db.session.merge(staff_top)
         for perm in ('view', 'edit'):
             for patient in (
-                patient_w_id, patient_x_id, patient_y_id, patient_z_id):
+                    patient_w_id, patient_x_id, patient_y_id, patient_z_id):
```
Comment (ivan-c) [inline tests/test_user.py]: Can you put `):` on its own line, at the same indentation as `for` to make it more obvious what's in the block and what's part of the loop definition

File: tests/test_user.py [hunk 10]
```diff
@@ -811,14 +813,15 @@ def test_deep_tree_check_role(self):
...
+                patient_w_id, patient_x_id, patient_y_id, patient_z_id
+            ):
                 assert staff_top.check_role(perm, other_id=patient)
 
         # mid level staff can view/edit all
         staff_mid = db.session.merge(staff_mid)
         for perm in ('view', 'edit'):
             for patient in (
-                patient_w_id, patient_x_id, patient_y_id, patient_z_id):
+                    patient_w_id, patient_x_id, patient_y_id, patient_z_id):
```
Comment (ivan-c) [inline tests/test_user.py]: Can you move the `):` to a new line, aligned with `for`?

File: tests/test_user.py [hunk 11]
```diff
@@ -845,15 +848,15 @@ def test_deep_tree_check_role(self):
         # top level staff can view all, edit none
         staff_top = db.session.merge(staff_top)
         for patient in (
-            patient_w_id, patient_x_id, patient_y_id, patient_z_id):
+                patient_w_id, patient_x_id, patient_y_id, patient_z_id):
```
Comment (ivan-c) [inline tests/test_user.py]: Same with these, see above

File: tests/test_user.py [hunk 12]
```diff
@@ -973,15 +976,16 @@ def test_subject_relationships(self):
...
         other_user = self.add_user(username='other@foo.com')
-        data = {'relationships':[{'user': TEST_USER_ID,
-                                  'has the relationship': 'partner',
-                                  'with': other_user.id},]
-               }
+        data = {'relationships': [{
+            'user': TEST_USER_ID,
+            'has the relationship': 'partner',
+            'with': other_user.id}, ]
+                }
```
Comment (ivan-c) [inline tests/test_user.py]: Can you fix the indentation (align with `data`), or combine it with `]` ie `'with': other_user.id}, ]}`?

**Pull request merged**